src/routes/(admin)/dashboard/+page.svelte

```
@@ -1,10 +1,17 @@
 <script lang="ts">
+	import { goto } from '$app/navigation';
+	import { page } from '$app/state';
+	import StateSelection from '$lib/components/StateSelection.svelte';
+	import TagsSelection from '$lib/components/TagsSelection.svelte';
```

ESLint

[error] 'TagsSelection' is defined but never used.

(@typescript-eslint/no-unused-vars)